src/Morphir/TypeScript/AST.elm

```diff
@@ -31,22 +37,47 @@ type Privacy
     | Private
 
 
+{-| Represents the path to a module nameSpace, for creating import statements or references
+-}
+type alias NamespacePath =
```

Comment:
How is this different from Morphir.IR.FQName? (Maybe the comment should tell me that)

Reply:
An FQName has a third component: the local name. Comment updated with a better explanation:
```
{-| Represents the path to a module. Used in various ways to produce either a path
to a module file, or a reference to that module's namespace. (eg in imports)

This has two components, the package path and the module path.
(Note: this is different from a Morphir Fully Qualified Name, which has three
components: package path, module path AND a local name).

-}
```

Reply:
OK, I'm happy with that. I wonder if a Morphir.IR.FQName with an empty localName component would work, but might also be less clear.